src/plugins/share/public/components/url_panel_content.tsx

```diff
@@ -394,4 +434,37 @@ export class UrlPanelContent extends Component<Props, State> {
       </EuiFormRow>
     );
   };
+
+  private renderUrlParamExtensions = (): ReactElement | void => {
+    if (!this.props.isEmbedded) {
+      return;
+    }
+
+    const checkboxes = [
+      ['topMenu', 'Top menu'],
```

Comment:
These should be internationalized with `i18n.translate` like here: https://github.com/elastic/kibana/pull/58435/files#diff-2fa41be73e54b8cb0480dfc410ef33dcR301